.github/workflows/actions-pipeline.yml

```diff
@@ -0,0 +1,70 @@
+name: Build
+
+concurrency:
+  group: ${{ github.ref }}
+  cancel-in-progress: false
+  
+on:
+  push:
+    branches: 
+      - 'main'
+      - 'release/*'
+  pull_request:
+    branches: 
+      - 'main'
+      - 'release/*'
+      - '!/docs/'
+      - '!/*.md'
+
+env:
+  RunPeVerify: true 
+  
+jobs: 
+
+  build: 
+    strategy:
+          matrix:
+            configuration: [ debug, release ]
+            projectloadstyle: [ net35, net40, net46, net6_0, net7_0 ]
+            os: [ windows-latest, ubuntu-latest ]
+            exclude: 
+              - projectloadstyle: net35
+                os: ubuntu-latest
+              - projectloadstyle: net40
+                os: ubuntu-latest
+              - projectloadstyle: net46
+                os: ubuntu-latest
+
+    env:
+          Solution_Name: Open-XML-SDK.sln  
+          
+    runs-on: ${{ matrix.os }}
+    
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+
+    - name: Install .NET Core
+      uses: actions/setup-dotnet@v3
+      with:
+          dotnet-version: 6.0.x
+
+    - name: Restore the Projects
+      run: dotnet restore --verbosity normal
+      env:
+          Configuration: ${{ matrix.configuration }}
```

Comment:
Can these be set as env block for the whole job?

Reply:
yes, done.